downstream/modules/platform/ref-automation-hub-requirements.adoc

```diff
@@ -32,8 +32,10 @@ If you install {PrivateHubname} from an internal address, and have a certificate
 To avoid this, use the `automationhub_main_url` inventory variable with a value like \https://pah.example.com linking to the {PrivateHubName} node in the installation inventory file.
 
 This adds the external address to `/etc/pulp/settings.py`.
-
 This implies that you only want to use the external address.
 
 For information on inventory file variables, see link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/{PlatformVers}/html/red_hat_ansible_automation_platform_installation_guide/appendix-inventory-files-vars[Inventory File Variables] in the _{PlatformName} Installation Guide_.
 ====
+
+include::../../../../modules/hub/ref-ha-hub-reqs.adoc[leveloffset=+1]
```

Comment:
This will either require a symlink or you can move the ref-ha-hub-reqs.adoc file into the modules/platform directory. Aine or I can help you with this depending on when you're ready.